docs/source/index.rst

```diff
@@ -100,6 +100,65 @@ Launch an interactive container to start using Morpheus:
 
 See :doc:`basics/overview` for more information on using the CLI.
 
+Building Localy
+---------------
+To get started, first clone the Morpheus repo:
+
+.. code-block:: bash
+
+   export PYTHON_VER=3.8
+   export RAPIDS_VER=22.04
```

Comment:
Probably should be 22.06 based on @cwharris 's upcoming changes.

Reply:
I checked, and that PR (#180) is targeting branch-22.08.
@cwharris do you know if moving to cudf=22.06 is going to be merged back into the 22.06 release?

Reply:
I had it targeting the wrong branch but it's merged in 22.06 now.

Reply:
Updated, @drobison00 & @cwharris thanks